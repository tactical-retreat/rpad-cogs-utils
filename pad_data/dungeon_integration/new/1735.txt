Hera Dragon Descended! - Darkness' Dragon-Annihilation

stage 0

825 - Godly Knight of the Sky, Verche @ level 20
Verche Dark Halven
Reduce damage from Dark by 50%
Sanction
Bind Dark cards for 10 turns
----- HP <= 100 -----
Conviction
92400 Damage - Change Jammer to Light, Deal 220% damage
----- HP <= 70 -----
Celestial Armor
Void damage >= 1000000 for 99 turns (1 time use)

stage 1

393 - Fertility Deity, Evil Ceres @ level 10
30% Resolve
Survive attacks with 1 HP when HP > 30%
Dark Melancholy + Divine Protection of the Good Harvest
Absorb combo <= 6 damage for 999 turns + Voids status ailments for 999 turns
----- Turn 1 -----
Sephiroth Anger
25704 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Heal, Deal 240% damage (1 time use)
----- HP <= 100 -----
Light Fascination
Absorb Light damage for 1 turns
----- HP <= 49 -----
Lucent Blessing + Sephiroth Rage
Light skyfall +20% for 5 turns + Change all orbs to Fire, Water, Wood, Light, Dark, Heal
----- HP <= 49 -----
Sephiroth Rage
36414 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Heal, Deal 340% damage
----- HP <= 29 -----
God Tree's Basket + Plant Fascination
Void damage >= 500000 for 99 turns + Absorb Wood damage for 1 turns
----- HP <= 29 -----
Flame Fascination + Natural Disaster
Absorb Fire damage for 1 turns + Change 5th row to Heal orbs (25% chance)
Ice Fascination + Natural Disaster
Absorb Water damage for 1 turns + Change 5th row to Heal orbs (25% chance)
Dark Fascination + Natural Disaster
Absorb Dark damage for 1 turns + Change 5th row to Heal orbs (25% chance)
Plant Fascination + Natural Disaster
Absorb Wood damage for 1 turns + Change 5th row to Heal orbs (25% chance)
----- HP <= 1 -----
Healing Wind + Evil Eye
Enemy recover 50% hp + Player -500% HP

stage 2

2720 - Darkness Goddess, Hera Dragon @ level 10
Black Body + Dark Ball + Darkdragon's Baptism
Voids status ailments for 999 turns + Absorb damage >= 2000000 damage for 999 turns + Enemy action (1 time use)
----- Turn 1 -----
Summon Vinus + Breath of Compassion
Do nothing + Player recover 100% hp (1 time use)
----- Turn 2 -----
Brilliant World
109850 Damage - Deal 1000% damage (2 hits, 500% each) (1 time use)
----- Turn 3 -----
Summon Mineva + Breath of Compassion
Do nothing + Player recover 100% hp (1 time use)
----- Turn 4 -----
Celestial Prison's Scorching Heat
137312 Damage - Deal 1250% damage (5 hits, 250% each) (1 time use)
----- Turn 5 -----
Summon Seres + Breath of Compassion
Do nothing + Player recover 100% hp (1 time use)
----- Turn 6 -----
Soul Burst
164775 Damage - Deal 1500% damage (3 hits, 500% each) (1 time use)
----- Turn 7 -----
Summon Neptunus + Breath of Compassion
Do nothing + Player recover 100% hp (1 time use)
----- Turn 8 -----
Abysstrom
193336 Damage - Deal 1760% damage (4 hits, 440% each) (1 time use)
----- Turn 9 -----
Summon Ades + Breath of Compassion
Do nothing + Player recover 100% hp (1 time use)
----- Turn 10 -----
Death Scythe
230685 Damage - Deal 2100% damage (6 hits, 350% each) (1 time use)
----- Execute below actions in order repeatedly -----
Gathering Magical Power
Do nothing
Dark Eyes + Ultra Graviton Breath
Voids player buff effects + Enemy action

