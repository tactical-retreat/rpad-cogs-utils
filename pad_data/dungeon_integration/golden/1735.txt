Hera Dragon Descended! - Darkness' Dragon-Annihilation

stage 0

825 - Godly Knight of the Sky, Verche @ level 20
Verche Dark Halven
Reduce damage from Dark by 50%
Sanction
Bind Dark cards for 10 turns
----- HP <= 100 -----
Conviction
92400 Damage - Change Jammer to Light, Deal 220% damage
----- HP <= 70 -----
Celestial Armor
Void damage >= 1000000 for 99 turns

stage 1

393 - Fertility Deity, Evil Ceres @ level 10
30% Resolve
Survive attacks with 1 HP when HP > 30%
Dark Melancholy + Divine Protection of the Good Harvest
Absorb combo <= 6 damage for 999 turns + Voids status ailments for 999 turns
----- Turn 1 -----
Sephiroth Anger
25704 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Heal, Deal 240% damage
----- HP <= 100 -----
Light Fascination
Absorb Light damage for 1 turns
----- HP <= 49 -----
Lucent Blessing + Sephiroth Rage
Light skyfall +20% for 5 turns + Change all orbs to Fire, Water, Wood, Light, Dark, Heal
----- HP <= 49 -----
Sephiroth Rage
36414 Damage - Change all orbs to Fire, Water, Wood, Light, Dark, Heal, Deal 340% damage
----- HP <= 29 -----
God Tree's Basket + Plant Fascination
Void damage >= 500000 for 99 turns + Absorb Wood damage for 1 turns
----- HP <= 29 -----
Flame Fascination + Natural Disaster
Absorb Fire damage for 1 turns + Change 5th row to Heal orbs
Ice Fascination + Natural Disaster
Absorb Water damage for 1 turns + Change 5th row to Heal orbs
Dark Fascination + Natural Disaster
Absorb Dark damage for 1 turns + Change 5th row to Heal orbs
Plant Fascination + Natural Disaster
Absorb Wood damage for 1 turns + Change 5th row to Heal orbs
----- HP <= 1 -----
Healing Wind + Evil Eye
Enemy recover 50% hp + Player -500% HP

stage 2

2720 - Darkness Goddess, Hera Dragon @ level 10
Black Body + Dark Ball + Darkdragon's Baptism
Voids status ailments for 999 turns + Absorb damage >= 2000000 damage for 999 turns + Enemy action
----- HP <= 100 -----
Gathering Magical Power
Do nothing
----- HP <= 100 -----
Dark Eyes + Ultra Graviton Breath
Voids player buff effects + Enemy action

